napari/layers/labels/_labels_key_bindings.py

```diff
@@ -90,6 +90,26 @@ def increase_label_id(layer: Labels):
     layer.selected_label += 1
 
 
+@register_label_action(
+    trans._("Decrease the paint brush size by one."),
+)
+def decrease_brush_size(layer: Labels):
+    """Decrease the brush size"""
+    if layer.brush_size > 1:  # here we should probably add a non-hard-coded
```

Comment:
Create variables `MIN_BRUSH_SIZE = 1` and MAX_BRUSH_SIZE = 40` and compare to them, instead of hardcoded variables. 

This is halfway to making it fully configurable because someone could modify them from code.